### PR TITLE
Feature/post add

### PR DESCRIPTION
**Pull request by Ayerbe88**
Integration of posting advertisement web component in Vue.

### CODE REVIEW

File: am-ui/webcomponents/create-ad-webcomponent/src/components/CreateAdFields.vue [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+        console.log(fieldValue);
+        errors.value[i] = true;
+        containsErrors = true;
+      } else if (fieldValue.toString().length < 1) {
+        errors.value[i] = true;
+        containsErrors = true;
+      }
+    }
+  });
+  console.log(errors.value);
```
Comment (stefanleminh) [inline am-ui/webcomponents/create-ad-webcomponent/src/components/CreateAdFields.vue:334]: Is this log needed?

File: data-service-backend/target/classes/api.yaml [hunk 2]
```diff
@@ -0,0 +1,405 @@
+openapi: 3.0.3
```
Comment (stefanleminh) [inline data-service-backend/target/classes/api.yaml:1]: What are the files in data-service-backend for? I thought these folders are not used anymore?

**Review by stefanleminh — CHANGES_REQUESTED** [review]
On top of the points mentioned in the comments, do we need the empty file form-field-mapper.js?

File: am-ui/webcomponents/create-ad-webcomponent/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Vue 3 + Vite
+
+This template should help get you started developing with Vue 3 in Vite. The template uses Vue 3 `<script setup>` SFCs, check out the [script setup docs](https://v3.vuejs.org/api/sfc-script-setup.html#sfc-script-setup) to learn more.
+
+## Recommended IDE Setup
+
+- [VSCode](https://code.visualstudio.com/) + [Volar](https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.volar)
```
[the remark this answers is not in the document]
Comment (Ayerbe88) [reply am-ui/webcomponents/create-ad-webcomponent/README.md:7]: Readme updated

File: am-ui/webcomponents/create-ad-webcomponent/src/components/CreateAd.ce.vue [hunk 4]
```diff
@@ -0,0 +1,449 @@
...
+$white: #fff;
+$matte-white: #f6f6f6;
+$grey: #232323;
+$light-grey: #868686;
+$akros-beige: #b0b0b0;
+$light-beige: #ece6df;
+$akros-red: #d60929;
+$akros-red-bg: #d6092812;
+$dark-red: #9c132c;
+/*                              RESET
```
[the remark this answers is not in the document]
Comment (Ayerbe88) [reply am-ui/webcomponents/create-ad-webcomponent/src/components/CreateAd.ce.vue:137]: Yea, you are right, I separated both the colors and the rest into separate files which are now imported in the styles of the CreateAd

File: am-ui/webcomponents/create-ad-webcomponent/src/components/CreateAdFields.vue (hunk 1 above)
Comment (Ayerbe88) [reply am-ui/webcomponents/create-ad-webcomponent/src/components/CreateAdFields.vue:334]: Removed along with other 2

**Pull request merged**